pages.hi/common/git-init.md

```diff
@@ -0,0 +1,11 @@
+# git init
+
+> एक नया स्थानीय गिट रिपॉजिटरी शुरू करता है। अधिक जानकारी: https://git-scm.com/docs/git-init ।
```

Comment:
The links should be between `<>` in `.md` files, (I'm not sure how to write the `।`):
```suggestion
> एक नया स्थानीय गिट रिपॉजिटरी शुरू करता है।
> अधिक जानकारी: <https://git-scm.com/docs/git-init>।
```